packages/documents/lib/resolve.js

```diff
@@ -76,7 +76,7 @@ const extractUserUrl = url => {
     return
   }
   return extractUserPath(
-    `${urlObject.path}${urlObject.hash || ''}`
+    `${urlObject.pathname}${urlObject.hash || ''}`
```

Comment:
even though it currently does not matter, let's keep the convention that everything after `hostname` is available for resolving / extracting:
```suggestion
    `${urlObject.pathname}${urlObject.search}${urlObject.hash}`
```